api/pom.xml

```diff
@@ -70,7 +70,13 @@
             <artifactId>fluent-hc</artifactId>
             <version>4.5.6</version>
         </dependency>
-        
+
+        <dependency>
```

Comment:
There is a conflict with jackson core 2.8.6 included as a dependency with orcid-api-client